yax/docs/src/main/tut/13-Unit-Testing.md

```diff
@@ -14,6 +14,7 @@ As with earlier chapters we set up a `Transactor` and YOLO mode. We will also us
 #+cats
 As with earlier chapters we set up a `Transactor` and YOLO mode. We will also use the `doobie-specs2-cats` and `doobie-scalatest-cats` add-ons.
 #-cats
+If you run into any difficulties it may be useful to refer to the [Working Around Bad Metadata Section](/docs/06-Checking.html)
```

Comment:
The link should be `06-Checking.html` without the leading `/docs/` … it's confusing, sorry.

Reply:
My bad, I'll fix that up.